src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/ServiceJourneys/Table/index.js

```diff
@@ -34,10 +35,10 @@ class ServiceJourneysTable extends Component {
       serviceJourneys.length > 0 ? (
         serviceJourneys.map((jp, i) => (
```

Comment:
Det är möjligt att ändra ordningen på journeypatterns, så hade varit nice att komma iväg från att använda index som key? 😊

Reply:
Prøver med JSON.stringify for nå 😅

Reply:
JourneyPatterns mener jeg kommer med ID så du kan bruke denne

Reply:
Ah, bra! Endrer.